netbox/dcim/choices.py

```diff
@@ -985,6 +986,7 @@ class InterfaceTypeChoices(ChoiceSet):
             (
                 (TYPE_100ME_FX, '100BASE-FX (10/100ME FIBER)'),
                 (TYPE_100ME_LFX, '100BASE-LFX (10/100ME FIBER)'),
+                (TYPE_100ME_FX_SFP, '100BASE-FX-SFP (10/100ME FIBER)'),
```

Comment:
The SFP interface type does not imply a specific medium (e.g. FX), just the line rate. Also, this needs to be moved to the "Ethernet (modular)" group as it is modular in nature.
```suggestion
                (TYPE_100ME_SFP, 'SFP (100ME)'),
```